installation/scripts/prow/jobs/provision-compass.sh

```diff
@@ -48,8 +48,10 @@ cp yq "$HOME/bin/yq" && cp yq "/usr/local/bin/yq"
 log::info "Successfully installed yq version: $YQ_VERSION"
 
 # Install Kyma to be later used in run.sh
+KYMA_CLI_VERSION="2.5.0"
 log::info "Installing Kyma CLI version: $KYMA_CLI_VERSION"
-KYMA_CLI_VERSION=$(cat ${COMPASS_SOURCE_DIR}/installation/resources/KYMA_VERSION)
+# TODO: Kyma 2.4.3 release exists, but Kyma CLI 2.4.3 does not
+# KYMA_CLI_VERSION=$(cat ${COMPASS_SOURCE_DIR}/installation/resources/KYMA_VERSION)
```

Comment:
Hmm, does that mean we'll use kyma CLI 2.5.0 locally as well? Will this cause any issues? And what are the pros/cons using 2.4.2 CLI?

Reply:
This specific script is used only during Prow jobs, the integration jobs for example. Locally (`run.sh`), your own version of the Kyma CLI is used. The Kyma CLI version is different from the actual Kyma release (`--source` during `kyma deploy`).

Personally, I use Kyma CLI version 2.15.1 while installing Kyma 2.3.0 or 2.4.3 and it does not show any issues; I doubt it would be a problem. I have not tested the Kyma CLI 2.4.2 as I thought it was a safer bet to go with 2.5.0.